pygdf/series.py

```diff
@@ -281,18 +281,51 @@ def _unaryop(self, fn):
     def __add__(self, other):
         return self._binaryop(other, 'add')
 
+    def __radd__(self, other):
+        return self.__add__(other)
+
     def __sub__(self, other):
         return self._binaryop(other, 'sub')
 
+    def __rsub__(self, other):
+        if isinstance(other, (int, float,
+                              np.int32, np.int64,
+                              np.float32, np.float64)):
+            empty = np.empty(len(self))
+            empty.fill(other)
+            other = Series(empty)
```

Comment:
I'm not sure this is the right approach, it doesn't handle if a user wants to do a pygdf.series.Series + numpy.array, pandas.Series, or any other array-like or series-like objects that we've handled elsewhere.

I think it would be better if we utilized the existing logic in https://github.com/gpuopenanalytics/pygdf/blob/master/pygdf/series.py#L262 which handles converting the scalar/array-like/series-like object into a pygdf Series. From a glance, I think we could change the line https://github.com/gpuopenanalytics/pygdf/blob/master/pygdf/series.py#L269 to be:
```
outcol = other._column.binary_operator(fn, self._column)
```

Reply:
This makes sense. Will implement the logic present in the existing function. Thanks for the advice!